airflow/config_templates/default_airflow.cfg

```diff
@@ -815,8 +815,8 @@ worker_container_image_pull_policy = IfNotPresent
 # If True, all worker pods will be deleted upon termination
 delete_worker_pods = True
 
-# If True (default), worker pods will be deleted only on task success
-delete_worker_pods_on_success = True
+# If False (default), failed worker pods will not be deleted so users can investigate them.
+delete_worker_pods_on_failure = True
```

Comment:
Just so I'm clear: if both of these delete_ options are set, then even failed pods will be deleted. Correct?

Reply:
Yes. I'm making the DEFAULT to be "delete only succeeded pods."